### DISCUSSION TITLE
Proposed Profile Pictures

### DISCUSSION THREAD

**Discussion by thisiscoding1234** [dispatch-d/Web#1] [Polls]
# Proposed Profile Picture Styles
## Why are we doing this?
We are doing this to see what style people prefer.

### Style 1
<img src="https://api.dicebear.com/9.x/glass/svg?seed=[Username]&radius=50&backgroundColor=4747eb,4762eb,477eeb,4799eb,47b4eb,47d0eb,47eb47,47eb62,47eb7e,47eb99,47ebb4,47ebd0,47ebeb,6247eb,62eb47,7e47eb,7eeb47,9947eb,99eb47,b447eb,b4eb47,d047eb,d0eb47,eb4747,eb4762,eb477e,eb4799,eb47d0,eb47eb,eb6247,eb7e47,eb9947,ebb447,ebd047,ebeb47,b6e3f4,c0aede,d1d4f9,ffd5dc,ffdfbf&backgroundType=gradientLinear&backgroundRotation=360,-360" style="width:25%; height:25%;"/>

### Style 2
<img src="https://api.dicebear.com/9.x/rings/svg?seed=[Username]&radius=50&backgroundType=gradientLinear&ringColor=4db6ac,81c784,9575cd,aed581,ba68c8,e57373,f06292,ff8a65,ffb74d,ffd54f,dce775,4dd0e1,7986cb&ringFive=full,half,quarter,eighth&ringFour=half,quarter,full,eighth&ringOne=half,quarter,full,eighth&ringThree=half,quarter,full,eighth&ringTwo=half,quarter,full,eighth&backgroundColor=c0aede,b6e3f4" style="width:25%; height:25%;"/>

**Comment by werdl** [comment]
I like style 1, perhaps with a D in the middle?

**Comment by Neodx2** [comment]
none of em
